pkg/pillar/types/newlogtypes.go

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2020 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package types
+
+import (
+	"time"
+)
+
+const (
+	// DevPrefix - device log file prefix string
+	DevPrefix = "dev.log."
+	// AppPrefix - app log file prefix string
+	AppPrefix = "app."
+	// AppSuffix - app log file suffix string, the appuuid is between the AppPrefix and AppSuffix
+	AppSuffix = ".log."
+)
+
+type logfileMetrics struct {
+	// from newlogd
+	NumGZipFilesSent  uint64 // total gzip files uploaded, device
+	NumGZipBytesWrite uint64 // total gzip log in bytes, device
+	NumBytesWrite     uint64 // total log bytes write to file before gzip, device
+	NumGzipFileInDir  uint32 // current number of gzip files remain, device
+	NumInputEvent     uint64 // total event input from log source, device include kmessages
+	// from loguploader
+	NumGZipFileRetry      uint64    // total gzip file upload retries, device
+	NumGZipFileDrop       uint32    // total gzip file upload 4xx failure and kept on device
```

Comment:
name has "Drop" but the comment says "kept". Confusing.

Reply:
Yes. Trying to figure out a good term. we will not send this anymore, and it's kept only on device;-) Maybe changing this name to "NumGZipFileKeptLocal"

Reply:
That name makes a lot more sense.